app/peerinfo/peerinfo.go

```diff
@@ -177,12 +180,17 @@ func (p *PeerInfo) sendOnce(ctx context.Context, now time.Time) {
 				return
 			}
 
+			name := p2p.PeerName(peerID)
+
+			// Validator git hash with regex.
+			if !gitHashMatch.Match([]byte(resp.GitHash)) {
+				log.Error(ctx, "Invalid peer git hash", nil, z.Str("peer", name))
```

Comment:
see logging guidelines, this should be a warn

Reply:
done